tests/drop_packets/test_drop_counters.py

```diff
@@ -388,3 +389,54 @@ def test_ip_pkt_with_exceeded_mtu(do_test, ptfadapter, setup, tx_dut_ports, pkt_
         do_test("L2", pkt, ptfadapter, ports_info, setup["neighbor_sniff_ports"])
     finally:
         L2_COL_KEY = RX_DRP
+
+
+def get_target_port(duthost):
```

Comment:
This is not aligned with the scenario we observed.
The issue just happened while the interface is admin up AND oper down.

Reply:
@kevinskwang , we will change the PR to bring down the interface by shutting down the fanout interface instead. Will that work?

Reply:
Sounds good